pnpm-workspace.yaml

```diff
@@ -12,4 +14,13 @@ catalog:
   rimraf: "^5.0.10"
   typescript: "~5.6.3"
   undici: "^5.28.4"
-  vitest: ~2.1.8
+  vitest: "~2.1.8"
+  vite: "^5.0.12"
+
+catalogs:
+  vite-plugin:
+    "@cloudflare/workers-types": "^4.20241205.0"
```

Comment:
Is this needed? Dependabot updates `@cloudflare/workers-types` across the repo every week or so

Reply:
Cool. Let's remove it.